### PR TITLE
Hover on player list

### PR DESCRIPTION
**Pull request by vitsalis**

### CODE REVIEW

**Comment by vitsalis** [comment]
game2

**Comment by dionyziz** [comment]
Please update the screenshot.

**Comment by vitsalis** [comment]
game3

File: static/script/game/view.js [hunk 1]
```diff
@@ -65,20 +65,27 @@ var GameView = {
...
     processCreatures: function( creatures ) {
         $( '.creature' ).remove();
         for ( var i = 0; i < creatures.length; ++i ) {
             var creature = creatures[ i ];
             if ( creature.hp > 0 ) {
-                var $user = $( '.playerList li[data-id=' + creature.userid + ']' );
+                var $user = GameView.findUser( creature.userid );
                 var username = $user.text();
-                var color = $user.find( 'span.bubble' ).attr( 'data-color' );
+                var color = GameView.findUserColor( $user );
```
Comment (dionyziz) [inline static/script/game/view.js:81]: nice

File: static/script/game/view.js [hunk 2]
```diff
@@ -111,6 +118,23 @@ var GameView = {
             $element.css( key, value * GameView.PIXEL_MULTIPLIER );
         } );
     },
+    highlightCreatures: function( user, addShadow ) {
+        var $creatures = $( '.creature' );
+        var userid = user.getAttribute( 'data-id' );
+
+        $creatures.each( function( index, value ) {
+            var $creature = $creatures.eq( index );
```
Comment (dionyziz) [inline static/script/game/view.js]: $( this )

File: static/script/game/view.js [hunk 3]
```diff
@@ -111,6 +118,23 @@ var GameView = {
...
+
+            if ( $creature.attr( 'data-userid' ) == userid ) {
```
Comment (dionyziz) [inline static/script/game/view.js]: Filter using selector instead of .each + attribute equality check.

File: static/script/game/view.js [hunk 4]
```diff
@@ -111,6 +118,23 @@ var GameView = {
...
+                if ( addShadow ) {
+                    $creature.addClass( 'highlight' );
+                }
+                else {
+                    $creature.removeClass( 'highlight' );
```
Comment (dionyziz) [inline static/script/game/view.js]: `.toggleClass( 'highlight', addShadow )`

**Pull request merged**